public/selfoss.webapp

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "selfoss",
+    "background_color": "grey",
+    "icons": [{
+        "src": "images/logo_big.png",
```

Comment:
There is even bigger logo in [_docs/resources/logo.png](https://github.com/SSilence/selfoss/blob/master/_docs/resources/logo.png)